apps/framework-cli/src/cli/commands.rs

```diff
@@ -38,10 +38,16 @@ pub enum Commands {
     // },
     /// Starts a local development environment to build your data-intensive app or service
     Dev {},
+    /// Start a remote environment for use in cloud deployments
+    Prod {},
```

Comment:
`next` call this `start`, BTW.

Reply:
hmmm. I think `start` would be misleading to developers unless the comment says start in production.